superset/assets/package.json

```diff
@@ -81,7 +81,7 @@
     "d3-tip": "^0.9.1",
     "datamaps": "^0.5.8",
     "datatables.net-bs": "^1.10.15",
-    "deck.gl": "^5.3.5",
+    "deck.gl": "^5.3.4",
```

Comment:
This is to align Kepler's deck.gl dep with Superset's. It's important as having 2 versions of deck.gl errors out.